cmd/kubeadm/app/apis/kubeadm/types.go

```diff
@@ -43,9 +43,9 @@ type MasterConfiguration struct {
 }
 
 type API struct {
-	AdvertiseAddresses []string
-	ExternalDNSNames   []string
-	Port               int32
+	BindAddress      string
```

Comment:
Add comments here.  We need to document everything in types.go.

"The address for the API server to listen on.  This will be where the API server serves a TLS secured endpoint.  The default of 0.0.0.0 says "listen on all addresses."